server/channels/store/sqlstore/scheduled_post_store.go

```diff
@@ -136,3 +137,104 @@ func (s *SqlScheduledPostStore) getMaxMessageSize() int {
 
 	return s.maxMessageSizeCached
 }
+
+func (s *SqlScheduledPostStore) GetScheduledPosts(beforeTime int64, lastScheduledPostId string, perPage uint64) ([]*model.ScheduledPost, error) {
```

Comment:
Nit: Naming on this is a bit ambiguous, if this to fetch posts that we expect to send, maybe call it `GetScheduledPostsToProcess`

Reply:
Renamed it to `GetPendingScheduledPosts`.